config/default/forms/app/pregnancy_danger_sign_follow_up.xml

```diff
@@ -176,10 +236,37 @@
           <text id="/pregnancy_danger_sign_follow_up/inputs/source_id:label">
             <value>Source ID</value>
           </text>
+          <text id="/pregnancy_danger_sign_follow_up/patient_age_in_years:label">
+            <value>Patient Age in Years</value>
+          </text>
+          <text id="/pregnancy_danger_sign_follow_up/patient_id:label">
+            <value>Patient ID</value>
+          </text>
+          <text id="/pregnancy_danger_sign_follow_up/patient_name:label">
+            <value>Patient Name</value>
+          </text>
+          <text id="/pregnancy_danger_sign_follow_up/patient_short_name:label">
+            <value>Patient Short Name</value>
+          </text>
+          <text id="/pregnancy_danger_sign_follow_up/patient_short_name_start:label">
+            <value>Patient Sort Name Start Label</value>
+          </text>
+          <text id="/pregnancy_danger_sign_follow_up/patient_uuid:label">
+            <value>Patient UUID</value>
+          </text>
+          <text id="/pregnancy_danger_sign_follow_up/pregnancy_uuid_ctx:label">
+            <value>Pregnancy UUID</value>
+          </text>
+          <text id="/pregnancy_danger_sign_follow_up/t_danger_signs_referral_follow_up:label">
+            <value>Danger Signs Present</value>
+          </text>
+          <text id="/pregnancy_danger_sign_follow_up/t_danger_signs_referral_follow_up_date:label">
+            <value>Danger Signs Follow Up Date</value>
+          </text>
```

Comment:
Are these calculations for labels? If so they should be removed, or moved to the `cht::notes` column.